SCClassLibrary/Common/GUI/Base/QWebView.sc

```diff
@@ -68,73 +188,141 @@ WebView : View {
+	}
 
-	// The arguments passed to the 'value' method are this WebView instance and
-	// a String for the requested URL to be reloaded.
+	onContentsSizeChanged_{
+		|func|
+		this.manageFunctionConnection( onContentsSizeChanged, func, 'contentsSizeChanged(QSizeF)' );
+		onContentsSizeChanged = func;
+	}
 
-	// If this method is called with nil argument, WebView's page reload handling will
-	// be restored again.
+	onAudioMutedChanged_{
+		|func|
+		this.manageFunctionConnection( onAudioMutedChanged, func, 'audioMutedChanged(bool)' );
+		onAudioMutedChanged = func;
+	}
 
-	onReload_ { arg func;
-		this.manageFunctionConnection( onReload, func, 'reloadTriggered(QString)' );
-		this.setProperty( \delegateReload, func.notNil );
-		onReload = func;
+	onRecentlyAudibleChanged_{
+		|func|
+		this.manageFunctionConnection( onRecentlyAudibleChanged, func, 'recentlyAudibleChanged(bool)' );
+		onRecentlyAudibleChanged = func;
 	}
 
-	onJavaScriptMsg_ { arg func;
-		this.manageFunctionConnection( onJavaScriptMsg, func,
-			'jsConsoleMsg(const QString&, int, const QString&)' );
+	onJavaScriptMsg_ {
+		|func|
+		this.manageFunctionConnection( onJavaScriptMsg, func, 'jsConsoleMsg(const QString&, int, const QString&)' );
 		onJavaScriptMsg = func;
 	}
 
-	enterInterpretsSelection { ^this.getProperty( \enterInterpretsSelection ); }
+	zoom					{ 			^this.getProperty('zoom') }
+	zoom_					{ |zoom| 	this.setProperty('zoom', zoom) }
 
-	enterInterpretsSelection_ { arg bool;
-		this.setProperty( \enterInterpretsSelection, bool );
-	}
+	hasSelection			{ 			^this.getProperty('hasSelection') }
+
+	selectedText			{ 			^this.getProperty('selectedText') }
 
-	editable { ^this.getProperty( \editable ); }
+	title					{ 			^this.getProperty('title') }
 
-	editable_ { arg bool;
-		this.setProperty( \editable, bool );
+	requestedUrl			{			^this.getProperty('requestedUrl') }
+
+	url						{ 			^this.getProperty('url') }
+	url_					{ |url| 	this.setProperty('url', url) }
+
+	enterInterpretsSelection { 			^this.getProperty('enterInterpretsSelection') }
+	enterInterpretsSelection_{ |b| 		this.setProperty('enterInterpretsSelection', b) }
+
+	editable				{ 			^this.getProperty('editable') }
+	editable_				{ |b| 		this.setProperty('editable', b) }
+
+	pageBackgroundColor		{ 			^this.getProperty('backgroundColor') }
+	pageBackgroundColor_	{ |color| 	this.setProperty('backgroundColor', color) }
+
+	contentsSize			{ 			^this.getProperty('contentsSize') }
+
+	scrollPosition			{ 			^this.getProperty('scrollPosition') }
+
+	audioMuted				{ 			^this.getProperty('audioMuted') }
+	audioMuted_				{ |muted| 	this.setProperty('audioMuted', muted) }
+
+	overrideNavigation		{ 			^this.getProperty('overrideNavigation') }
+	overrideNavigation_		{ |b| 		this.setProperty('overrideNavigation', b) }
+
+	back 					{ this.triggerPageAction(\back) }
+	forward					{ this.triggerPageAction(\forward) }
+	stop					{ this.triggerPageAction(\stop) }
+	reload					{ this.triggerPageAction(\reload) }
```

Comment:
this is really ridiculous looking with tabs at 8 spaces; could you please reformat it to use tabs only for leading whitespace, and single spaces otherwise?

Reply:
<img width="745" alt="image" src="https://user-images.githubusercontent.com/15369640/40207238-7f8ce32a-5a01-11e8-9cc5-2b748a36e343.png">